Sources/Sentry/SentryAppStateManager.m

```diff
@@ -54,7 +54,8 @@ - (SentryAppState *)buildCurrentAppState
                                              osVersion:UIDevice.currentDevice.systemVersion
                                               vendorId:vendorId
                                            isDebugging:isDebugging
-                                   systemBootTimestamp:self.sysctl.systemBootTimestamp];
+                                   systemBootTimestamp:self.sysctl.systemBootTimestamp
+                                        timezoneOffset:[NSTimeZone localTimeZone].secondsFromGMT];
```

Comment:
`m`: We could add `secondsFromGMT` to `SentryCurrentDateProvider` to make this easier to test.

Reply:
I did, but I'm not sure what else to change to "make this easier to test"?